### DISCUSSION TITLE
Statistics Training Effects + Training Performance graphs

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1294] [General]
In MyTourBook 19.6 Training Effect was introduced.  However I cannot populate the Statistics with any values.
>Training Effects + Training Performance columns
> Reimport training values (from .fit + .sml files)
> Training Effects + Training Performance graphs in the new training statistics

What is needed for these Statistics?

![Image1](https://github.com/mytourbook/mytourbook/assets/106929478/b6a770dc-ae13-4803-8e78-427350287a25)

**Comment by FJBDev** [comment]
@stvzldn 
Did you make sure that the files you import in MT contain `Training Effect` data ?

**Reply by stvzldn** [reply]
Looks like my Edge 520 does not report this data.  But still a working device with good battery life.  Maybe something new next year.

**Reply by FJBDev** [reply]
Indeed, a quick Google search shows that the Training Effect is not supported on the Edge 520 [as of today](https://forums.garmin.com/sports-fitness/sports-fitness/f/edge-520-520-plus/229613/edge-520-plus-training-effect)
